spec.md

```diff
@@ -491,6 +499,30 @@ The foundation block simply sequences operations and other blocks
 }
 ```
 
+### Parallel quantum block
+
+A grouping of quantum operations to be performed in parallel
+
+```json5
+{
+  "block": "parallel",
+  "ops": [{...}, ...],
+  "metadata": {...}  // Optional
+}
+```
+
+Example:
+
+```json5
+{
+  "block": "parallel",
+  "ops": [{"qop": "RZ", "angles": [[1.5], "pi"], "args": [["q", 0], ["q", 1]]},
+          {"qop": "RZ", "angles": [[1.0], "pi"], "args": [["q", 2], ["q", 3]]},
+          {"qop": "RZ", "angles": [[0.5], "pi"], "args": [["q", 4], ["q", 5]]}
+          ]
+}
+```
+
```

Comment:
Also, add a note either here or while introducing the example before the code block such as: "the six gate applications here are interpreted to be applied in parallel".